wscl-issues/draft/boolean-definition

```diff
@@ -0,0 +1,76 @@
+Issue:          BOOLEAN-DEFINITION
+Forum:          Cleanup
+Category:       CLARIFICATION
+Status:         draft
+Edit History:   16-May-24, Version 1 by Jin-Cheng Guu
+References:     BOOLEAN
+
+Problem Description:
+
+  In the draft ANSI Common Lisp specification, the description of the type
+  BOOLEAN [a] can be more accurate.
+
+    [a]: The type boolean contains the symbols t and nil, [..]
+
+  It is more accurate to use the word "consists of" than "contains". After
+  all, according to the glossary [b], BOOLEAN is exclusively either the symbol
+  T or the symbol NIL.
+
+    [b]: boolean n. an object of type boolean; that is, one of the following
+    objects: the symbol t (representing true), or the symbol nil (representing
+    false).
+
+Proposal: (BOOLEAN-DEFINITION:TYPE-BOOLEAN-IS-T-OR-NIL)
+
+  This proposal changes the description of the type BOOLEAN so that [a]
+  instead reads:
+
+    > The type boolean consists of the symbols t and nil, [..]
+
+Test Cases:
+
+(defun one ()
+    (assert
+     (and
+      (equal '(t t)
+             (multiple-value-list (subtypep 'boolean '(member t nil))))
+      (equal '(t t)
+             (multiple-value-list (subtypep '(member t nil) 'boolean))))))
+
+  (one) => NIL
```

Comment:
Please put the output in a comment and enclose the error in square brackets. We are working on a system that will auto-update the "Current Practices" section based on the "Test Cases" section.

Reply:
@yitzchak I'm not sure what you mean. I believe what I did here is the same as in, say, [adjust-array-displaced-to-constraints](https://github.com/s-expressionists/wscl/blob/main/wscl-issues/draft/adjust-array-displaced-to-constraints).

Reply:
Change line 49 to `(one) ; => NIL`. We are in the process of changing the formatting of the test cases. Not all issues have been updated.